src/scm/bitbucket.js

```diff
@@ -62,131 +64,136 @@ class Bitbucket {
         data.parents = parent;
       }
       $.ajax({
-        url: `${this.baseUrl}/repositories/${repo}/src`,
-        headers: {
-          'Authorization': `Bearer ${this.accessToken}` 
-        },
-        contentType: 'application/x-www-form-urlencoded',
-        method: 'POST',
-        crossDomain: true,
-        traditional: true,
-        data: data,
-      })
-      .then(resolve)
-      .fail(reject);
+          url: `${this.baseUrl}/repositories/${repo}/src`,
+          headers: {
+            'Authorization': `Bearer ${this.accessToken}`
+          },
+          contentType: 'application/x-www-form-urlencoded',
+          method: 'POST',
+          crossDomain: true,
+          traditional: true,
+          data: data,
+        })
+        .then(resolve)
+        .fail(reject);
     });
   }
 
-  push(code){
+  push(code) {
     const changed = $('.diff-file:checked').toArray().map(elem => elem.value);
     const files = changed.filter(f => code.gas[f]).map(f => {
-      return { name: f, content: code.gas[f] }
+      return {
+        name: f,
+        content: code.gas[f]
+      }
     });
     const deleteFiles = changed.filter(f => !code.gas[f]);
     const comment = $('#commit-comment').val();
 
     this.commitFiles(context.repo.fullName, context.branch, null, files, deleteFiles, comment)
-    .then(() => {
-      showAlert(`Successfully push to ${context.branch} of ${context.repo.fullName}`);
-    })
-    .catch((err) => {
-      showAlert('Failed to push', LEVEL_ERROR);
-    });
+      .then(() => {
+        showAlert(`Successfully push to ${context.branch} of ${context.repo.fullName}`);
+      })
+      .catch((err) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/scm/bitbucket.js#L97"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>